bin/lint_localized_strings_format

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -eu
+
+# Attempts to generate the frozen `.strings` file to gets sent to GlotPress
+# after a new release code freeze.
```

Comment:
Does `genstrings` emit warnings? Or errors only? The manpage has no mentions of "error" and only one of "warnings", yet the output we filter for is `error:`. 🤔 

```suggestion
# Attempts to generate the frozen `.strings` file—which usually gets sent to GlotPress
# after a new release code freeze—to catch potential errors early on.
```

I'm mostly asking because, if it does also print warnings, it might be useful to catch them too (and treat them as errors if possible).

Reply:
That is a very good question… that I don't have the answer to 😅